CMakeLists.txt

```diff
@@ -123,7 +136,7 @@ set(DOC_INPUT_ROOT_DIRS_LIST
   ${CMAKE_SOURCE_DIR}/VulkanExperiment/src 
   ${CMAKE_SOURCE_DIR}/VulkanExperiment/include 
   ${CMAKE_SOURCE_DIR}/Werld/src 
-  ${CMAKE_SOURCE_DIR}/Werld/include 
+  # ${CMAKE_SOURCE_DIR}/Werld/include 
```

Comment:
Whats this bad boiii? Werld doesnt have any external include files, i should probably just delete this, since werld is entirely an application